backend/src/lib.rs

```diff
@@ -150,6 +150,11 @@ impl LTN {
         Ok(serde_json::to_string(&self.map.filters_to_gj()).map_err(err_to_js)?)
     }
 
+    #[wasm_bindgen(js_name = renderTurnRestrictions)]
+    pub fn render_turn_restrictions(&self) -> Result<String, JsValue> {
```

Comment:
Not a problem, but trying to understand: 

You use the term `render` in some of these methods - does it connote something in particular? (vs. say `get_turn_restrictions`)

Reply:
Hmm, no. Looks like a habit I just retained from widgetry days. If the intent is only for visualization, I tend to use it. I wouldn't mind switching to `get` more consistently.